### ISSUE TITLE
Camera ready version

### ISSUE THREAD

**Issue by hbell99** [zhouxian/FluidLab#1]
Hi! Thanks for your great work!

I am wondering whether the camera ready version will be released?

**Comment by zhouxian** [comment]
Thanks for asking! The paper camera ready deadline is 1st Mar. So should be around that time.

**Comment by glue25** [comment]
Sorry to bother, I would like to ask, when will the camera ready version be released? Thank you!

**Comment by zhouxian** [comment]
Apologies. Thing have been a bit hectic recent weeks. I'll try to have it released by the end of next week.

**Comment by glue25** [comment]
I appreciate your excellent work very much, I am very sorry to bother you again, but I still want to ask when the camera ready version will be released? Thank you!

**Comment by glue25** [comment]
I sincerely appreciate your outstanding work, and I apologize for any inconvenience caused by my inquiry. May I respectfully request to know the expected release date of the camera-ready version? Thank you for your kind assistance.

**Comment by zhouxian** [comment]
@glue25 Hi! My deep apologies. I am fighting with multiple deadlines this month and am deeply sorry for the delay. I have finally started on this. Should be out by next Friday.

**Comment by DanielTakeshi** [comment]
To clarify is this the camera-ready for ICLR? I see this on arXiv:
https://arxiv.org/abs/2303.02346
Dated March 04. Isn't this the finalized version?

**Comment by DanielTakeshi** [comment]
@zhouxian Just checking, is this issue outdated?

**Comment by zhouxian** [comment]
@DanielTakeshi Yes this is the released camera ready version. However, there's still some tasks remaining that I haven't got a chance to fully clean up, so I am leaving this issue open.
